src/pyorcidator/helper.py

```diff
@@ -41,81 +51,98 @@ def get_external_ids(data) -> Mapping[str, str]:
         url = d["url"]["value"].rstrip("/")
         for key, url_prefix in PREFIXES:
             if url.startswith(url_prefix):
-                rv[key] = url[len(url_prefix):]
+                rv[key] = url[len(url_prefix) :]
     return rv
 
 
-def render_orcid_qs(orcid):
+def render_orcid_qs(orcid: str) -> str:
     """
     Import info from ORCID for Wikidata.
 
     Args:
         orcid: The ORCID of the researcher to reconcile to Wikidata.
     """
-    data = get_orcid_data(orcid)
+    return "\n".join(line.get_line() for line in get_orcid_quickstatements(orcid))
+
 
-    with open("sample.json", "w+") as f:
-        f.write(json.dumps(data, indent=2, sort_keys=True, ensure_ascii=False))
+def _get_orcid_qualifier(orcid: str) -> Qualifier:
```

Comment:
Although the base class is called Qualifier, in this case you're using it to generate a quickstatements reference/source (Sxx). Maybe calling it `_get_orcid_reference` would be more appropriate? Just a matter of opinion though.

Reply:
Not in a way that would reflect in the data model you implemented, at least not that I can see. Just that quickstatements uses a different syntax - Sxx vs Pxx - for sources and qualifiers. https://www.wikidata.org/wiki/Help:QuickStatements#Add_statement_with_qualifiers

As I said, more of a small nitpick while I'm looking at the code :P

Reply:
so one possibility is to create more specific data models that have more specific regular expressions. If you think this would be valuable then I totally support it! One advantage of having more specific stuff is you can write more careful documentation and tests (not the sexy part of programming, but for sure the most important)